.eslintrc.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
 		'computed-property-spacing': [ 1, 'always' ],
 		'func-call-spacing': 1,
 		indent: [ 1, 'tab', { SwitchCase: 1 } ],
+		'jsx-a11y/accessible-emoji': 1,
+		'jsx-a11y/click-events-have-key-events': 1,
+		'jsx-a11y/heading-has-content': 1,
+		'jsx-a11y/href-no-hash': 1,
+		'jsx-a11y/html-has-lang': 1,
+		'jsx-a11y/iframe-has-title': 1,
+		'jsx-a11y/img-has-alt': 1,
+		'jsx-a11y/label-has-for': 1,
```

Comment:
This is the only rule I have any problem with, but perhaps I just need to be educated.

I can see how this makes sense for:

```html
<label>Email:</label>
<input type="email">
```

But I thought it was perfectly valid to do this:

```html
<label>Email: <input type="email"></label>
```

But it errors with this rule. Am I wrong?

(Reason I'm so disinclined is because working with IDs in React is very ugly)